hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/ClusterStateResponse.java

```diff
@@ -104,6 +110,7 @@ private ClusterStateResponse(Builder b) {
     this.containers = b.containers;
     this.missingContainers = b.missingContainers;
     this.openContainers = b.openContainers;
+    this.deletedContainers = b.deletedContainers;
```

Comment:
currently we have a TestTotalOpenContainerCount and TestTotalDeletedContainerCount test classes for each different state counts we can consolidate all of them into a single test class that calculates the container count of different states.
We could call that class TestContainerStateCounts